pkgs/tools/graphics/mscgen/default.nix

```diff
@@ -12,7 +24,7 @@ stdenv.mkDerivation {
     sha256 = "3c3481ae0599e1c2d30b7ed54ab45249127533ab2f20e768a0ae58d8551ddc23";
   };
 
-  buildInputs = [ flex bison gd libjpeg libpng freetype zlib libwebp ];
+  buildInputs = [ bison flex freetype gd libjpeg libpng libwebp pkg-config zlib ];
```

Comment:
bison, flex and pkg-config should be in `nativeBuildInputs`.

Reply:
o.O :)